wolfhsm/wh_client.h

```diff
@@ -559,6 +559,57 @@ void wh_Client_SetKeyRsa(RsaKey* key, whNvmId keyId);
  * @param[in] keyId Key ID to be associated with the AES key.
  */
 void wh_Client_SetKeyAes(Aes* aes, whNvmId keyId);
+
+/**
+ * @brief Runs the Cmac operation in a single call with a wolfHSM keyId.
+ *
+ * This function does entire cmac operation in one function call with a key
+ * already stored in the HSM. This operation evicts the key from the HSM cache
+ * after the operation though it will still be in the HSM's NVM if it was
+ * commited
+ *
+ * @param[in] cmac Pointer to the CMAC key structure.
+ * @param[out] out Output buffer for the CMAC tag.
+ * @param[out] outSz Size of the output buffer.
+ * @param[in] in Input buffer to be hashed.
+ * @param[in] inSz Size of the input buffer.
+ * @param[in] keyId ID of the key inside the HSM.
+ * @param[in] heap Heap pointer for the cmac struct.
+ * @return int Returns 0 on success, or a negative error code on failure.
+ */
+int wh_Client_AesCmacGenerate(Cmac* cmac, byte* out, word32* outSz,
```

Comment:
could you add a blurb to the draft manual in the crypto section of `chapter05-client-library.md mentioning that for CMAC functions they should use the client API wrappers instead of wolfCrypt directly if you want to use hardware keys